src/widget/tool/profileimporter.cpp

```diff
@@ -0,0 +1,65 @@
+#include "profileimporter.h"
+#include <QString>
+#include <QFile>
+#include <QDir>
+#include <QFileDialog>
+#include <QFileInfo>
+#include "src/persistence/settings.h"
+#include "src/core/core.h"
+#include "src/widget/gui.h"
+#include <QDebug>
+#include <QMessageBox>
+
+ProfileImporter::ProfileImporter(QWidget *parent) : QWidget(parent)
+{
+
+}
+
+bool ProfileImporter::importProfile()
+{
+    QString path = QFileDialog::getOpenFileName(this,
```

Comment:
@agilob Remember our first discussion? :smile:

@PKEv This is not a bug and totally correct. It's a wrong assumption, that the system style is used, just because a widget has no parent. This is simply untrue and setting parent to 0 introduces other effects we do not want. We can only preserve the system's style by actually correctly assigning it to the widget. If we want to use system default, the css file has to change like:

```css
!QFileDialog, !QDialog { ... }
```

Like I wrote before, it is better to implement a `QStyle` class.

Or we just stick with qTox style, which is also fine and recently was agreed by "the team".